### PR TITLE
very basic docker instructions and files

### PR DESCRIPTION
**Pull request by Greg-Griffith**
very basic docker instructions and files for those who want to test a specific build that failed travis on their local machine

### CODE REVIEW

**Review by sickpig — APPROVED** [review]

File: docker/README.md [hunk 1]
```diff
@@ -42,3 +42,15 @@ example -p 8333:1234 will map the port 8333 internal to the docker container to
 
 ## To connect to the container in a bash shell
 `docker exec -it <container name> bash`
+
+
+
+
+
+
```
Comment (sickpig) [inline docker/README.md:50]: nit: too much spaces? (once rendered they'll go away, still)

**Review by sickpig — APPROVED** [review]
re-approve after last commit which add a docker composer

File: docker/README.md (hunk 1 above)
Comment (Greg-Griffith) [reply docker/README.md:50]: idk man, does that really matter? =p

Comment (sickpig) [reply docker/README.md:50]: it doesn't :smile:

this why I label it as a nitpick and why I approved the PR regardless of my comment.

**Pull request merged**